### PR TITLE
Add more missing translations in templates

### PR DESCRIPTION
**Pull request by pyrog**
See last comment of #498

### CODE REVIEW

File: ember/app/templates/flights.hbs [hunk 1]
```diff
@@ -4,7 +4,7 @@
       {{t "flights"}}
 
       {{#if loading}}
-        <small>Loading...</small>
+        <small>{{t loading}}</small>
```
Comment (Turbo87) [inline ember/app/templates/flights.hbs]: I think this should be `{{t "loading"}}`, right?

Comment (pyrog) [inline ember/app/templates/flights.hbs]: Yes, thank you to notice that.

**Comment by pyrog** [comment]
cleanup and rebasing of all my translations 😄

**Comment by Turbo87** [comment]
Thanks! 👍

**Pull request merged**